src/app/about/about.scss

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
extra space at the top.

Reply:
Maybe this can be added to the linter.  I will fix this.

Reply:
@immangat , that's a good question.

Do you want to create a issue ticket for this.

We want to find linter for scss.